Gemfile

```diff
@@ -9,4 +9,4 @@ gemspec
 # your gem to rubygems.org.
 
 # Load Gemfile with dependencies from manageiq
-eval_gemfile(File.expand_path("spec/manageiq/Gemfile", __dir__))
+eval_gemfile(File.expand_path('spec/manageiq/Gemfile', __dir__))
```

Comment:
I would be careful modifying files that come from the common generator.  It makes follow-up re-generation more difficult by increasing the diff that the author needs to review.